### DISCUSSION TITLE
Difference between Wflow_sbm and Wflow_hbv for snow and glacier melting

### DISCUSSION THREAD

**Discussion by mullerrobert799** [Deltares/Wflow.jl#291] [Q&A] [answered]
I am wondering to find out what are the major differences between Wflow_sbm and Wflow_hbv? Documentation said both can be used for snow and glacier melting but what is a preferred model for the modelling of snow and glacier?

**Comment by JoostBuitink** [comment, marked as answer]
Both hydrological models use the same concept to simulated the snow and glacier processes. 

The difference between wflow_sbm and wflow_hbv is in the conceptualization of all other processes. The concept behind wflow_sbm is much more physically based, while the concept behind wflow_hbv is much more conceptually based. Determining which concept is most suitable for your region highly depends on the question you want to answer, the available data to build and calibrate/validate your model. Typically wflow_hbv requires calibration to get realistic hydrological behavior, while wflow_sbm (assuming you build the model using HydroMT-Wflow) gives a relatively accurate first-order simulation due to the pedo-transfer functions.
